jdi-light-html-tests/src/test/java/io/github/epam/example/complex/DropdownExampleTests.java

```diff
@@ -58,7 +58,6 @@ public void checkValuesTest() {
 
     @Test
     public void dropdownTestSize() {
-        colors2.expand();
         colors2.assertThat().notEmpty();
         colors2.assertThat().size(5);
     }
```

Comment:
Please ensure you have a blank line at the end of all files you modified

Reply:
have a blank line